sympy/matrices/matrices.py

```diff
@@ -1270,24 +1270,60 @@ def eigenvals(self, error_when_incomplete=True, **flags):
     def eigenvects(self, error_when_incomplete=True, iszerofunc=_iszero, **flags):
         """Return list of triples (eigenval, multiplicity, basis).
 
-        The flag ``simplify`` has two effects:
+        Parameters
+        ==========
+
+        error_when_incomplete : bool, optional
+            Raise an error when not all eigenvalues are computed. This is
+            caused by ``roots`` not returning a full list of eigenvalues.
+
+        iszerofunc : function, optional
+            Specifies a zero testing function to be used in ``rref``.
+
+            Default value is ``_iszero``, which uses naive and fastest approach
+            by using sympy's default assumption handler.
+
+            It can also accept any user-specified zero testing function, if it
+            is formatted as a function which accepts a single symbolic argument
+            and returns ``True`` if it is tested as zero and ``False`` if it
+            is tested as non-zero, and also ``None`` if it is undecidable.
+
+        simplify : bool or function, optional
             1) if bool(simplify) is True, as_content_primitive()
             will be used to tidy up normalization artifacts;
+
             2) if nullspace needs simplification to compute the
             basis, the simplify flag will be passed on to the
             nullspace routine which will interpret it there.
 
-        Parameters
-        ==========
+        chop : bool or positive number, optional
+            If the matrix contains any Floats, they will be changed to Rationals
+            for computation purposes, but the answers will be returned after
+            being evaluated with evalf. If it is desired to removed small
+            imaginary portions during the evalf step, pass a value for the
+            ``chop`` flag.
 
-        error_when_incomplete : bool
```

Comment:
@sidhantnagpal 
`multiple` is being ignored in `eigenvects` for now.
```
flags.pop('multiple', None)  # remove this if it's there
```

I think `eigenvects` does not follow any form of return values like in `roots` or `eigenvals`, though I don't know if it can be an issue.

Reply:
Yes, it seems that `multiple` is already documented for `eigenvals`. This would not be required for `eigenvects` unless there's good reason to have similar functionality for it.